domains/server.alpha.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "YouFoundAlpha",
+        "email": "sweepalf@gmail.com"
+    },
+    "record": {
+       "CNAME": "serveo.net",
```

Comment:
CNAME record cannot be used in conjunction with other records and vise versa.

Reply:
Does is-a.dev support serveo tunnelling service?

Reply:
No.